caddyconfig/httpcaddyfile/builtins.go

```diff
@@ -703,6 +703,8 @@ func parseRedir(h Helper) (caddyhttp.MiddlewareHandler, error) {
 	// don't redirect non-browser clients
 	if code != "200" {
 		hdr = http.Header{"Location": []string{to}}
+	} else {
+		hdr = http.Header{"Content-Type": []string{"text/html; charset=utf-8"}}
```

Comment:
Actually, I think we could probably move this to inside the `case "html"` switch. Best to consolidate that logic in one place.

Reply:
Done :+1: